src/v2/coins/abstractUtxoCoin.js

```diff
@@ -517,41 +513,78 @@ class AbstractUtxoCoin extends BaseCoin {
   }
 
   /**
-   * Indicates whether a coin supports pay-to-witness script hash addresses
+   * Indicates whether a coin supports pay-to-witness-script-hash outputs wrapped in pay-to-script-hash outputs
+   * @returns {boolean}
+   */
+  supportsP2shP2wsh() {
+    return false;
+  }
+
+  /**
+   * Indicates whether a coin supports pay-to-witness-script-hash outputs
    * @returns {boolean}
    */
   supportsP2wsh() {
     return false;
   }
 
   /**
+   * TODO(BG-11487): Remove addressType, segwit, and bech32 params in SDKv6
    * Generate an address for a wallet based on a set of configurations
-   * @param addressType
-   * @param keychains Array of objects with xpubs
-   * @param threshold Minimum number of signatures
-   * @param chain Derivation chain
-   * @param index Derivation index
-   * @param segwit
-   * @param bech32
+   * @param addressType {string}   Deprecated
+   * @param keychains   {[object]} Array of objects with xpubs
+   * @param threshold   {number}   Minimum number of signatures
+   * @param chain       {number}   Derivation chain (see https://github.com/BitGo/unspents/blob/master/src/codes.ts for
+   *                                                 the corresponding address type of a given chain code)
+   * @param index       {number}   Derivation index
+   * @param segwit      {boolean}  Deprecated
+   * @param bech32      {boolean}  Deprecated
```

Comment:
Kinda sad that we had the parameter and are deprecating it before it ever got used in production. :hushed: